oeps/processes/oep-0021-proc-deprecation.rst

```diff
@@ -275,15 +277,54 @@ Do the following to document your proposal:
       **Deprecation** section. See Deprecated_ for considerations.
    #. If automated migration will be needed, explain your migration plan in the
       **Migration** section.
-   #. If there is any additional publicly shareable information or data from
-      your earlier analysis, include that in the **Additional Info** section.
+   #. **Additional Info**:
+
+      - If there is any additional publicly shareable information or data from
+        your earlier analysis, include that in the **Additional Info** section.
+      - If you know who plans to coordinate the ticket, include the
+        name in the **Additional Info** section. The deprecation
+        ticket is not active until there is a coordinator.
 
 #. Check that your ticket appears on the `DEPR Project Board`_.
 
 .. note::
   You can create a ticket even if you don't yet have all the information
-  gathered yet. You simply must wait until you have completed the issue before
-  moving on to communicating.
+  gathered yet. You simply must wait until you have completed the issue and
+  have a coordinator before moving on to the formal **Communication** step.
+
+(Optional) Start a preliminary discussion
+-----------------------------------------
+If you want to start a preliminary discussion or seek an owner for the
+deprecation ticket, you can post a message in the `Development Discussion
+Board`_ and any appropriate channels in the `Open edX slack`_.  This allows
+you to start a discussion and ask for feedback before somebody commits to
+coordinating the DEPR process.
+
+.. _Development Discussion Board: https://discuss.openedx.org/c/development/11
+
+Claim the DEPR Ticket as Coordinator
+------------------------------------
+A DEPR ticket is not active until it has a coordinator.
+
+The coordinator is responsible for:
+
+- Moving the deprecation work forward
+- Following the DEPR workflow phases
+- Informing the community about any changes
+- Communicating if they are unable to continue coordinating the DEPR
+
+When you are ready to coordinate the ticket, post a comment on the
+GitHub issue saying you're doing so, and mark yourself as the GitHub
+issue's owner.
```

Comment:
```suggestion
When you are ready to coordinate the ticket, post a comment on the
GitHub issue saying you're doing so, and mark yourself as the GitHub
issue's owner.  This can be done even if you don't have write access to 
the ticket by making a comment on the ticket that says ``assign me``.
```

Not required suggestion to help people who don't have access to assign tickets but want to participate.